src/app/services/environments.service.ts

```diff
@@ -437,10 +437,16 @@ export class EnvironmentsService {
         routeResponse = this.schemasBuilderService.buildRouteResponse();
       }
 
+      const prefix = this.store.getActiveEnvironment().endpointPrefix;
+      let endpoint = log.url.slice(1); // Remove the initial slash '/'
+      if (prefix.length && endpoint.startsWith(prefix)) {
```

Comment:
I think it would be safer to test `prefix` directly instead of `prefix.length` in case it's null or undefined (even if it shouldn't) But no need to worry, I will do this small modification, and also check why the tests are not running on macOS.